AsyncDisplayKit/ASTableView.mm

```diff
@@ -467,29 +477,36 @@ - (CGSize)rangeControllerViewportSize:(ASRangeController *)rangeController
 - (void)rangeController:(ASRangeController *)rangeController didInsertNodesAtIndexPaths:(NSArray *)indexPaths withAnimationOption:(ASDataControllerAnimationOptions)animationOption
 {
   ASDisplayNodeAssertMainThread();
-
-  [super insertRowsAtIndexPaths:indexPaths withRowAnimation:(UITableViewRowAnimation)animationOption];
+  ASPerformBlockWithoutAnimation(animationOption == UITableViewRowAnimationNone, ^{
+    [super insertRowsAtIndexPaths:indexPaths withRowAnimation:(UITableViewRowAnimation)animationOption];
+  });
 }
 
 - (void)rangeController:(ASRangeController *)rangeController didDeleteNodesAtIndexPaths:(NSArray *)indexPaths withAnimationOption:(ASDataControllerAnimationOptions)animationOption
 {
   ASDisplayNodeAssertMainThread();
 
-  [super deleteRowsAtIndexPaths:indexPaths withRowAnimation:(UITableViewRowAnimation)animationOption];
+  ASPerformBlockWithoutAnimation(animationOption == UITableViewRowAnimationNone, ^{
```

Comment:
Personally I read this fine, but for extra clarity I'd suggest either adding extra parentheses (make it more clear that this is evaluating an expression inside an arguments list) or create a local variable like "preventAnimation ="